src/modm/driver/display/ili9341_impl.hpp

```diff
@@ -350,6 +350,22 @@ Ili9341<Interface, Reset, Backlight, BufferSize>::drawImageRaw(glcd::Point upper
 	}
 }
 
+template <class Interface, class Reset, class Backlight, std::size_t BufferSize>
+void
+Ili9341<Interface, Reset, Backlight, BufferSize>::drawRaw(glcd::Point upperLeft,
+		uint16_t width, uint16_t height, glcd::Color* data)
+{
+	BatchHandle h(*this);
+
+	uint16_t* buffer = (uint16_t*)data;
+	for(size_t i = 0; i < uint16_t(width*height); i++) {
```

Comment:
There is an overflow bug. This is a 320x240 display driver. Thus, `width*height` can overflow a `uint16_t`.
```suggestion
	for(size_t i = 0; i < size_t(width*height); i++) {
```